tests/e2e/specs/nominate-owner.spec.js

```diff
@@ -0,0 +1,62 @@
+describe(
+  "Nominate Owner",
+  {
+    retries: {
+      runMode: 5,
+      openMode: 0,
+    },
+  },
+  () => {
+    before(() => {
+      cy.getMetamaskWalletAddress().then((address) => {
+        if (address !== "0xe0A71284EF59483795053266CB796B65E48B5124") {
+          cy.switchMetamaskAccount(1);
+        }
```

Comment:
able to do a simple mapping of account to address?

Reply:
erm, i dont understand what u mean by this, maybe can elaborate further abit?

Reply:
some sort of key value map. currently, mentally have to associate account 1 and 2 are to which addresses. theres some same vars in `setup-contracts` too

do we need that condition to check if address is not account 1 then switch to account 1?

Reply:
yup, we need the condition to check, this will reduce the fail rate.

And added a utils for the key value map for the account addresses.